docs/components/modeler/web-modeler/advanced-modeling/manage-connector-templates.md

```diff
@@ -0,0 +1,103 @@
+---
+id: manage-connector-templates
+title: Manage connector templates
+description: Manage your connector templates in Web Modeler.
+---
+
+export const UploadIcon = () => <span style={{verticalAlign: "text-top"}}><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24" width="16" height="16"><path d="M0 0h24v24H0z" fill="none"></path><path d="M9 16h6v-6h4l-7-7-7 7h4zm-4 2h14v2H5z" fill="currentColor"></path></svg></span>;
+
+<span class="badge badge--cloud">Camunda Platform 8 only</span>
+
+You can create and manage [connector templates](/components/integration-framework/connectors/custom-built-connectors/connector-templates.md) just as any other asset in a Web Modeler project.
+
+## Create connector templates
+
+You can create a new connector template using the following steps:
+
+1. Navigate to the project of your choice and click **New**.
+
+2. Choose to create a blank **Connector Template**.
+
+   ![Creating a new template](img/connector-templates/create-connector-template-1.png)
+
+3. You will be taken to the **connector template editor** screen. In this screen, you can define the connector template by writing the template JSON. The template editor supports you in writing the template by providing autocompletion, error highlighting and a live preview.
+
+   The anatomy of the editor interface is as follows:
+
+   - In the **breadcrumbs bar**, you can rename your template by clicking the chevron next to the template name. Note that you cannot change the name of your template in the template json, but only with this action.
+   - On the left, you see the **template JSON editor**. Here, you describe the actual template specification. The specification follows the [most recent element template schema](https://github.com/camunda/element-templates-json-schema). The editor is divided into two sections: a read-only section, containing the schema reference, the template name, and the template id, and a editable section, where you can [write your template specification](/components/modeler/desktop-modeler/element-templates/defining-templates.md).
```

Comment:
Consider using the same wording as we do in element template docs ([ref](http://localhost:3000/docs/next/components/modeler/desktop-modeler/element-templates/defining-templates/)):

```suggestion
   - On the left, you see the **template JSON editor**. Here, you define the actual template descriptor. The descriptor follows the [most recent element template schema](https://github.com/camunda/element-templates-json-schema). The editor is divided into two sections: a read-only section, containing the schema reference, the template name, and the template id, and a editable section, where you can [define your template descriptor](/components/modeler/desktop-modeler/element-templates/defining-templates.md).
```